Makefile

```diff
@@ -11,20 +11,24 @@ UTIL=Util/*.cpp
 
 EXECUTABLE=smashbot
 
-all: goals strats tactics chains main util
+all: util goals strats tactics chains main
```

Comment:
Ah yeah I wasn't sure about that. I tried it with -j12 and I guess it happened to work. What's a better approach?
Also `clean` should probably be adjusted to remove the old Constants.h.